docs/users.rst

```diff
@@ -1170,3 +1170,70 @@ stored in koji build, in section ``build.extra.operator_manifests.appregistry``.
 
 Manifests will not be pushed to the application registry for scratch builds,
 isolated builds, or re-builds to prevent unwanted changes
+
+
+Building Source Container Images
+================================
+
+OSBS is able to build source container image from a particular koji build
+previously created by OSBS. To crate a source container build you have to
+specify either koji N-V-R or koji build ID.
```

Comment:
How about
... koji build ID for the image build you want to create a source container image for.